Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-plp-images/index.jsx

```diff
@@ -1,8 +1,8 @@
-import React, {useEffect} from 'react'
+import React, { useEffect } from 'react'
 import PropTypes from 'prop-types'
 import DynamicImage from '@salesforce/retail-react-app/app/components/dynamic-image'
 
-const CloudinaryPlpImage = ({cloudinaryImage = {}, dynamicImageProps = {}, image = {}}) => {
+const CloudinaryPlpImage = ({ cloudinaryImage = {}, dynamicImageProps = {}, image = {} }) => {
     if (typeof window !== 'undefined' && cloudinaryImage.url) {
```

Comment:
no replacement of the analytics tag

Reply:
We are replacing the analytic tag directly in the image source at line 22 because if we replace the tag outside of the image source, it first renders the image source with the old analytic tag which is coming from the backend, so to overcome this issue we are directly replacing the tag within the image source.